pkg/compactor/blocks_cleaner.go

```diff
@@ -110,6 +123,14 @@ func NewBlocksCleaner(
 			Help: "Total number of blocks failed to be deleted.",
 		}),
 		blocksMarkedForDeletion: blocksMarkedForDeletion,
+		cleanerVisitMarkerReadFailed: promauto.With(reg).NewCounter(prometheus.CounterOpts{
+			Name: "cortex_compactor_cleaner_visit_marker_read_failed_total",
+			Help: "Total number of cleaner visit marker file failed to be read.",
+		}),
+		cleanerVisitMarkerWriteFailed: promauto.With(reg).NewCounter(prometheus.CounterOpts{
+			Name: "cortex_compactor_cleaner_visit_marker_write_failed_total",
+			Help: "Total number of cleaner visit marker file failed to be written.",
+		}),
```

Comment:
We have existing metrics like these in compactor. But tbh I don't feel they are that useful. What's the impact and action to take when these metrics increase?
Not blocking and we can keep them for now. We can think about this and revisit in the future

Reply:
I am ok to remove those metrics. Most of failure would come from bucket store operations and there is already metric to track that.

Reply:
Let's keep them for now.

Reply:
I removed those metrics to make the code clean. If we really need them, can add them back later.